sarracen/interpolate/interpolate.py

```diff
@@ -191,6 +189,14 @@ def _verify_columns(data: 'SarracenDataFrame',  # noqa: F821
     if data.hcol is None:
         raise KeyError("Smoothing length column does not exist in the "
                        "provided dataset.")
+    if data.mcol is None:
+        if 'mass' not in data.params:
+            raise KeyError("'mass' column does not exist in the "
+                           "provided dataset.")
```

Comment:
We can simplify this to
`if data.mcol is None and 'mass' not in data.params:`

Reply:
Also let's update the error message, since it could either be the column or params key that is missing. Here is the same error message that is used in calc_density() inside SarracenDataFrame.
`Missing particle mass in this SaracenDataFrame`